### ISSUE TITLE
[doesn't block release] Be able to provide snapshot so other people can import pre-release libraries

### ISSUE THREAD

**Issue by hackergil** [SAP/cloud-commerce-spartacus-storefront#438]
Provide snapshots/nightly builds of our libs for the community - not necessarily in npmjs.com.

**Comment by tobi-or-not-tobi** [comment]
nightly: regardless of quality, just the latest version at a given time
snapshots: the latest stable snapshot (based green pipeline)

I think we like snapshots. 

I've seen a common usage of @org/library-build. The additional complication with this approach, is the dependencies; they should be loaded from similar paths.

**Comment by hackergil** [comment]
@tobi-or-not-tobi I could not find @org/library-build. Can you elaborate on it and your ideas for this? Thanks.

**Comment by tobi-or-not-tobi** [comment]
@hackergil I tried to make an example :) the `-build` seem to be a convention (I read about it at https://blog.lacolaco.net/post/how-to-use-angular-2-nightly-builds/), but i'm not opinionated on how we do it.

**Comment by hackergil** [comment]
Blocked until we get the github repos for nightly builds

**Comment by KateChuen** [comment]
Split the tests into 2 directories: 
Smoke & Regression. 
In Smoke, we'll have these tests: 
- big-happy-path
- homepage
- login
- product-search
- routing
- language (when migrated)
- currency (when migrated)
In regression, all the rest

**Comment by hackergil** [comment]
The following repos are now setup:

https://github.com/SAP/cloud-commerce-spartacus-storefront-core-builds
 
https://github.com/SAP/cloud-commerce-spartacus-storefront-styles-builds
 
https://github.com/SAP/cloud-commerce-spartacus-storefront-storefront-builds

**Comment by hackergil** [comment]
PR [here](https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/2028)
Doc PR [here](https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/2029)

**Comment by KateChuen** [comment]
Don't have access to the snapshot repos yet. So can't test it yet.

**Comment by KateChuen** [comment]
Getting the latest snapshots works:
 

> "@spartacus/core": "SAP/cloud-commerce-spartacus-storefront-core-builds",
 "@spartacus/styles": "SAP/cloud-commerce-spartacus-storefront-styles-builds",
 "@spartacus/storefront": "SAP/cloud-commerce-spartacus-storefront-storefront-builds"


But getting specific snapshots doesn't work.

> "@spartacus/core": "SAP/cloud-commerce-spartacus-storefront-core-builds+ce88323eab813671a053860ef7122d5d612934a7",
"@spartacus/styles": "SAP/cloud-commerce-spartacus-storefront-styles-builds+ce88323eab813671a053860ef7122d5d612934a7",
"@spartacus/storefront": "SAP/cloud-commerce-spartacus-storefront-storefront-builds+ce88323eab813671a053860ef7122d5d612934a7",

**Comment by KateChuen** [comment]
Tested on dev2. 
- verified that the following gets the latest snapshots
> "@spartacus/core": "SAP/cloud-commerce-spartacus-storefront-core-builds",
"@spartacus/styles": "SAP/cloud-commerce-spartacus-storefront-styles-builds",
"@spartacus/storefront": "SAP/cloud-commerce-spartacus-storefront-storefront-builds"

- verified that we can also get specific snapshots. To test this, I picked one specific commit, checked that it's displayed. Then picked an older commit, and made sure the feature isn't displayed.
![specificSnapshot](https://user-images.githubusercontent.com/45409515/58353312-27a87600-7e3c-11e9-8427-f0a0bd044dbd.png)


Specific snapshot:
> "@spartacus/core": "SAP/cloud-commerce-spartacus-storefront-core-builds#core-0.1.0+47988098f",
"@spartacus/styles": "SAP/cloud-commerce-spartacus-storefront-styles-builds#styles-0.1.0+47988098f",
"@spartacus/storefront": "SAP/cloud-commerce-spartacus-storefront-storefront-builds#storefront-0.1.0+47988098f",
![currentSnapshot](https://user-images.githubusercontent.com/45409515/58353286-1c554a80-7e3c-11e9-94e9-b9eb38725154.png)


Older snapshot:
> "@spartacus/core": "SAP/cloud-commerce-spartacus-storefront-core-builds#core-0.1.0+114187375",
"@spartacus/styles": "SAP/cloud-commerce-spartacus-storefront-styles-builds#styles-0.1.0+114187375",
"@spartacus/storefront": "SAP/cloud-commerce-spartacus-storefront-storefront-builds#storefront-0.1.0+114187375",
![olderSnapshot](https://user-images.githubusercontent.com/45409515/58353304-22e3c200-7e3c-11e9-95d7-46dcdfb774cc.png)
